src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/IssueNumberRetriever.java

```diff
@@ -23,7 +27,18 @@ public IssueNumberRetriever(DataSource dataSource) {
     public long getNextIssueNumber() {
         try {
             Statement statement = dataSource.getConnection().createStatement();
-            ResultSet resultSet = statement.executeQuery(GET_NEXTVALUE_FROM_ISSUE_NUMBER_SEQUENCE);
+            ResultSet resultSet = statement.executeQuery(GET_NEXT_VALUE_FROM_ISSUE_NUMBER_SEQUENCE);
+            resultSet.next();
+            return resultSet.getLong("issue");
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+    
+    public long getCurrentIssueNumber() {
+        try {
+            Statement statement = dataSource.getConnection().createStatement();
+            ResultSet resultSet = statement.executeQuery(GET_CURRENT_VALUE_FROM_ISSUE_NUMBER_SEQUENCE);
             resultSet.next();
             return resultSet.getLong("issue");
         } catch (SQLException e) {
```

Comment:
Same as above. You can avoid wrapping checked exceptions into `RuntimeException` by adding Lombok [`@SneakyThrows`](https://projectlombok.org/features/SneakyThrows.html) annotation at method declaration. This way the code has less boilerplates and is more readable.